### DISCUSSION TITLE
have turbo-ignore use local copy of turbo when running its command

### DISCUSSION THREAD

**Discussion by arnorhs** [vercel/turborepo#8124] [Ideas]
### Goals

1. allows you to use the currently installed version of turbo, and not downloading a package
2. ensures that there won't be an accidental version mismatch

### Non-goals

1. i don't care about vercel

### Background

When running turbo-ignore in eg. a github action, it would be preferable if the build step, that already has cached and validated dependencies, including turbo and turbo-ignore.. and npx might not be in the current $PATH

### Proposal

I propose not using a global `npx turbo` command internally to run the turbo command - possibly simply overriding the executable, but i'm sure there's a better way to infer the turbo command from the local project scope

I'm referring to this line in particular:
https://github.com/vercel/turbo/blob/main/packages/turbo-ignore/src/ignore.ts#L107

**Comment by anthonyshew** [comment]
Thanks for this feedback. We made some changes for this behavior here: https://github.com/vercel/turbo/releases/tag/v2.0.3

The correct version should now be being inferred, according to what your repository is meaning to intend.
